superchain/init.go

```diff
@@ -89,9 +69,8 @@ func init() {
 			}
 			superchainEntry.ChainIDs = append(superchainEntry.ChainIDs, chainConfig.ChainID)
 			OPChains[chainConfig.ChainID] = &chainConfig
-			Addresses[chainConfig.ChainID] = &addrs
-			GenesisSystemConfigs[chainConfig.ChainID] = &genesisSysCfg
-
+			Addresses[chainConfig.ChainID] = &chainConfig.Addresses
+			GenesisSystemConfigs[chainConfig.ChainID] = &chainConfig.Genesis.SystemConfig
```

Comment:
We could file an issue describing an obvious cleanup to remove these mappings (breaking change).

Reply:
Filed issue here: https://github.com/ethereum-optimism/superchain-registry/issues/391